### PR TITLE
Enable Codecov in addition to Coveralls

### PR DESCRIPTION
**Pull request by nalimilan**
Couldn't figure how to enable Coveralls status indicators in pull requests.

### CODE REVIEW

**Comment by nalimilan** [comment]
I guess we need to merge this before we can see the status indicators? Yet I've enabled the Codecov App on GitHub...

**Comment by rofinn** [comment]
Yeah, I usually just accept that I need to push to master before confirming that my changes work for things like codecov and documentation.

**Comment by ararslan** [comment]
Enabling Codecov should make it show up in PRs even without a .codecov.yml, so long as there's been a Codecov submission. I wonder if there's something wrong with this repo itself if neither Coveralls nor Codecov are working as expected...?

**Comment by ararslan** [comment]
Is it even possible to turn off Codecov comments anymore? Anything I've tried in the past several months hasn't worked. Personally I prefer Coveralls but if it's not going to cooperate then I guess we don't really have a choice if we want to track coverage in PRs.

**Comment by rofinn** [comment]
Could we support both and maybe even keep the coveralls badge? I've been finding that while codecov has better github integration I find the website less reliable (random 500 errors).

**Comment by ararslan** [comment]
We can definitely have both

File: .travis.yml [hunk 1]
```diff
@@ -16,5 +16,5 @@ git:
 #  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
 #  - julia -e 'Pkg.clone(pwd()); Pkg.build("StatsBase"); Pkg.test("StatsBase"; coverage=true)'
 after_success:
-  - julia -e 'cd(Pkg.dir("StatsBase")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
-  - julia -e 'Pkg.add("Documenter"); cd(Pkg.dir("StatsBase")); include(joinpath("docs", "make.jl"))'
+  - julia -e 'cd(Pkg.dir("StatsBase")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Codecov.process_folder())'
```
Comment (rofinn) [inline .travis.yml]: I believe you can just do `Codecov.submit(process_folder())`.

**Comment by nalimilan** [comment]
I've changed the PR to enable both Codecov and Coveralls. Should we merge?

File: README.md [hunk 2]
```diff
@@ -8,6 +8,7 @@
 - **Build & Testing Status:**
   [![Build Status](https://travis-ci.org/JuliaStats/StatsBase.jl.svg?branch=master)](https://travis-ci.org/JuliaStats/StatsBase.jl)
   [![Coverage Status](https://coveralls.io/repos/JuliaStats/StatsBase.jl/badge.svg?branch=master)](https://coveralls.io/r/JuliaStats/StatsBase.jl?branch=master)
+  [![Coverage Status](http://codecov.io/github/JuliaStats/StatsBase.jl/coverage.svg?branch=master)](http://codecov.io/github/JuliaStats/StatsBase.jl?branch=master)
```
Comment (rofinn) [inline README.md]: Is it worth having 2 coverage badges?

Comment (ararslan) [inline README.md]: Yeah, why not

Comment (rofinn) [inline README.md]: It just seems redundant, but I guess it's convenient to have a link to both services.

Comment (ararslan) [inline README.md]: Most packages that have both services enabled display both badges, including the main Julia repo

**Pull request merged**